src/components/movies/MoviesEditForm.js

```diff
@@ -0,0 +1,92 @@
+import { useState, useEffect } from "react";
+import { useParams, useNavigate } from "react-router-dom";
+
+import { getMedia, updateMedia } from "../../api/fetch";
+
+import "../styles/MediaForm.css";
+
+export default function MoviesForm() {
+  const { id } = useParams();
+  const navigate = useNavigate();
+
+  const [movieTitle, setMovieTitle] = useState(null);
+
+  const [movie, setMovie] = useState({
+    type: "",
+    title: "",
+    country: "",
+    dateAdded: "",
+    description: "",
+    duration: "",
+    listedIn: "",
+    rating: "",
+    releaseYear: "",
+  });
+
+  useEffect(() => {
+    getMedia("movies", id)
+      .then((response) => {
+        setMovieTitle(response.title);
```

Comment:
this doesn't entirely work with the control component. Upon editing a field, it is currently clearing out all the other fields

Reply:
Will fix this over the weekend.